docs/development/reference/encryption-technical.mdx

```diff
@@ -6,17 +6,17 @@ sidebar_position: 5
 description: Technical Overview of Encryption Employed with Meshtastic from Firmware 2.5.0 Onwards
 ---
 
-## PSK for Channels, PKC for Direct Messages (DMs) and Admin Messages
+## PSK for Channels, PKC for Direct Messages and Admin Messages
 
-Starting with firmware version 2.5.0, which introduces Public Key Cryptography (PKC) for Direct Messages (DMs) and additionally incorporating the use of Session IDs for Admin Messages, the security architecture of Meshtastic has reached a new level of robustness.
+Starting with firmware version 2.5.0, which introduces Public Key Cryptography (PKC) for Direct Messages (DMs) and also incorporates the use of Session IDs for Admin Messages, the security architecture of Meshtastic has reached a new level of robustness.
 
-Below is a detailed overview of how PSK, PKC, and Session IDs are integrated into your system to secure Channels, Direct Messages, and Admin Messages.
+Below is a detailed overview of how Pre-Shared Key (PSK), PKC, and Session IDs are integrated into your system to secure Channels, DMs, and Admin Messages.
```

Comment:
I think there's a sentence missing here, along the lines of:
"Up to version 2.4.x of the firmware, Meshtastic relied entirely on Pre-Shared Key (PSK) cryptography for all messaging."

This serves 3 purposes - it name-checks PSK, it makes it clear that PSK was there all along, and it gets the full tl:dr in the first paragraph.